### DISCUSSION TITLE
Are the `open` and `close` methods on SlimSelect designed to be public?

### DISCUSSION THREAD

**Discussion by StevenEWright** [brianvoe/slim-select#485] [Q&A]
I see that the `SlimSelect` class has public `open` and `close` methods, but they aren't documented on the website, so I was just wondering if these are intended to be part of the supported public interface or if using them is potentially problematic for some reason I don't understand yet.

Thanks!

https://github.com/brianvoe/slim-select/blob/8a105ca2102a5e600d19daf3c25ff12714604d1e/src/slim-select/index.ts#L305

<img width="224" alt="image" src="https://github.com/brianvoe/slim-select/assets/484872/0a9d9a02-8300-46e0-8d8b-daf8601a9e8c">

**Comment by brianvoe** [comment]
Fixed will be added in next release.
